developer_manual/general/devenv.rst

```diff
@@ -12,8 +12,7 @@ Basic Tools
 ===========
 
 To be able to develop with ownCloud and also run unit tests, you will need to install `Node.js <https://nodejs.org>`_.
-
-Other required tools, `such as the required PHP modules`_, will be automatically installed by composer.
+Other required tools, `such as the required PHP modules`_, can be installed using composer.
```

Comment:
you will need to install node js and the required PHP apache modules. all other dependencies will be installed automatically using composer.

@IljaN told me that composer does not install the required modules, you have to do it manually.

Reply:
Ok, I'll do some digging.